app/controllers/application_controller.rb

```diff
@@ -25,6 +25,12 @@ def raise_not_found
     redirect_to '/404'
   end
 
+  rescue_from Rack::Timeout::RequestTimeoutException, with: :raise_timeout
+  
+  def raise_timeout
+    redirect_to "/timeout"
```

Comment:
This is more of the Rails way:

```suggestion
    redirect_to timeout_error_path
```